freqtrade/rpc/telegram.py

```diff
@@ -873,7 +877,9 @@ async def _profit(self, update: Update, context: CallbackContext) -> None:
                 f"*{'First Trade opened' if not timescale else 'Showing Profit since'}:* "
                 f"`{first_trade_date}`\n"
                 f"*Latest Trade opened:* `{latest_trade_date}`\n"
-                f"*Win / Loss:* `{stats['winning_trades']} / {stats['losing_trades']}`"
+                f"*Win / Loss:* `{stats['winning_trades']} / {stats['losing_trades']}`\n"
+                f"*Winrate:* `{winrate:.2f}%`\n"
```

Comment:
this could be compresse into one line `Win / Loss: 26 / 178 (12.75%)` to reduce overall linecount.

I'm not entirely sure if that makes sense for the /profit output though - which is not overly long anyway.